src/paho/mqtt/__init__.py

```diff
@@ -1,4 +1,10 @@
-__version__ = "1.2.2"
+__version__ = "1.3.0.dev0"
```

Comment:
I agree. There isn't a problem for paho here (right now), but it is _technically_ bad practice.

I think there's also an argument that, if it is defined as an attribute, then it should be in the top-level module (`paho`) rather than in `paho.mqtt`.